appveyor.yml

```diff
@@ -0,0 +1,29 @@
+build: false
+
+environment:
+    PYTHON: "C:\\Python27"
+    PYTHON_VERSION: "2.7.8"
+    PYTHON_ARCH: "32"
+
+    CONAN_REFERENCE: "Libgeohash/2469d3c"
+    CONAN_USERNAME: "ebclark2"
+    CONAN_CHANNEL: "testing"
+    VS150COMNTOOLS: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\Common7\\Tools\\"
```

Comment:
This var VS150COMNTOOLS is no longer needed

Reply:
Fixed